### ISSUE TITLE
Midpoint Review - IA Feedback - MHV Medications Prescription Grouping

### ISSUE THREAD

**Issue by erinrwhite** [department-of-veterans-affairs/va.gov-team#97435]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should: represent older prescriptions not as children of current prescriptions** I understand this is a wicked problem, but noting it here and ccing @sterkenburgsara @mnorthuis who were also on the call and have surfaced this issue with the model here.

  tl;dr: How can old/previous/expired prescriptions be represented in the information model as **peers** of current/active prescriptions, rather than **children** of them? Right now, the architecture is thus...
    - Current prescription
      - Previous prescription 1
      - Previous prescription 2
      - Previous prescription 3  

  ...but we know that it's really more like this (and they all use the same view/content template):
  - Current prescription
  - Previous prescription 1  
  - Previous prescription 2 
  - Previous prescription 3

  A correct, but **very long-paged** solution is to list all prescriptions on one page together. Another potential solution might be to list each prescription (including the current one) as a child page of the medication page. 

  Getting this content model nailed down will make links to prescriptions far less brittle (i.e. changing based on their status), and will help maintain a consistent programmatic approach for linking to prescriptions.

  Y'all indicated you're also seeking to understand how much fidelity/information should be displayed for past prescriptions, and how far back we might want to go with this list. Thanks for grappling with this, and please lean on Sara as your resident content modeling wizard for feedback or ideas. Thank you!

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
